packages/txservice/src/transaction.ts

```diff
@@ -276,7 +281,9 @@ export class Transaction {
       const result = await this.provider.confirmTransaction(response, undefined, 60_000 * 20);
       if (result.isErr()) {
         // No errors should occur during this confirmation attempt.
-        throw result.error;
+        throw new TransactionServiceFailure("Unable to confirm transaction even after receiving 1 confirmation.", {
+          error: result.error,
+        });
```

Comment:
should include hash as a separate json field

Reply:
kk